x/farming/spec/01_concepts.md

```diff
@@ -1,35 +1,49 @@
 <!-- order: 1 -->
 
  # Concepts
+
 ## Farming Module
 
-`x/farming` is a Cosmos SDK module that implements farming functionality that keeps track of the staking and provides farming rewards to farmers. A primary use case is to use this module to provide incentives for liquidity pool investors for their pool participation. 
+The `x/farming` Cosmos SDK module implements farming functionality that keeps track of staking and provides farming rewards to farmers. A primary use case of this module is to provide incentives for liquidity pool investors for their pool participation. 
 
 ## Plans
 
-There are two types of farming plans in the `farming` module as below.
+There are two types of farming plans in the `farming` module:
+
+### Public Farming Plan
 
-### 1. Public Farming Plan
+A public farming plan can be created only through governance proposal. The proposal must be first agreed and passed before a public farming plan can be created. 
```

Comment:
is there a creation fee for public farming plan? if not, let's specify that here

Reply:
That's great point. No, there is no creation fee for public farming plan. Since public plan proposal needs to go through governance process, we design the spec to have a creation fee for private farming plan only.